lib/tasks/cip_seed_dump.rake

```diff
@@ -2,5 +2,5 @@
 
 desc "Dump CIP content into a file"
 task cip_seed_dump: :environment do
-  system "bundle exec rake db:seed:dump FILE=db/seeds/cip_seed_dump.rb EXCLUDE='[:created_at, :updated_at]' MODELS='LeadProvider, CourseYear, CourseModule, CourseLesson' IMPORT=true"
+  CoreInductionProgrammeExporter.new.run
```

Comment:
The old task did not want to fire from controller in production env :(

Reply:
Nice and clean